asm/macros/event.inc

```diff
@@ -262,6 +262,13 @@
 	.2byte \output
 	.2byte SPECIAL_\function
 	.endm
+	
+	@ temporary solution
+	.macro specialvar_ output, functionId
```

Comment:
What's this for?

Reply:
oh, it was something I tried to do before the data was dumped and forgot to remove it